### DISCUSSION TITLE
Problem between pandoc and libreoffice

### DISCUSSION THREAD

**Discussion by Arnaud71100** [jgm/pandoc#10440] [Q&A]
Good morning
I am French so what you are reading (if anyone is!) is the result of an automatic translation.
I have a problem with pandoc. I just did an update (change from 0.1.1.1-128 -> 3.1.11.1-9) and since then, when I open a libreoffice document (.odt or .ods), pandoc launches automatically and consumes all the memory of my computer. This is what I read when I look at the current tasks:
`pandoc -s -t markdown -- /home/arnaud/mondocument.odt`
Does anyone understand why?
Thank you in advance!
Arnaud

**Comment by jgm** [comment]
That's quite bizarre, and I don't think it has anything to do with pandoc itself.  However, it's difficult to diagnose this without being able to see more about your system.  Are you opening the file by clicking on an icon in a file manager?  Perhaps your OS is somehow set up to run pandoc instead of libreoffice when you click on an .odt?

The other thing that is strange about what you report is "update...from 0.1.1.1-128 -> 3.1.11.1-9".  Update of what, exactly?  Your OS?  Libreoffice?  Pandoc?  Pandoc never had a 0.1.1.1 release, so I assume it's not pandoc.

**Comment by badumont** [comment]
Although the update seems to be responsible for this change, this has nothing to do with Pandoc itself, but with how your computer determines what to do when you open a file. Pandoc itself does not have the power to set this. You should turn to your distribution's forum or to the maintainer of the pandoc package to investigate the problem.

**Comment by Arnaud71100** [comment]
Thank you very much for your comments. Thanks to these I understood that the problem does not come from pandoc but from the application that I use to navigate in my files and launch/open them (the application in question is **ranger**). When I use another application to open a Libre Office document, the problem does not appear.
Thank you again for your answers which were very useful to me!
And hello from France!!
